templates/user-research/index.html

```diff
@@ -0,0 +1,230 @@
+      </div>
+      <div class="col-9 col-medium-4">
+        <p class="p-heading--2 p-block"><strong>Help make Canonical products better.</strong><br>Share your thoughts and experiences.</p>
+        <p class="p-block">You'll get early access to the products Canonical's engineers and designers are working on right now &ndash; we love to collaborate with users when creating new features and designs. Your insights will influence our direction, and everyone's user experience.</p>
+        <p class="p-block"><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp" class="p-button--positive">Join our User Research Program</a></p>
+          {{ image (
+            url="https://assets.ubuntu.com/v1/e549fdc0-Canonical-AI-ML-Illustrations-v5-03.png",
+            alt="",
+            width="1200",
+            height="898",
+            hi_def=True,
+            loading="auto"
+            ) | safe
+          }}
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width">
+    <div class="row p-block">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-muted-heading">Current studies</h2>
+      </div>
+      <div class="col-9 col-medium-4">
+        <p class="p-heading--2">Navigating ubuntu.com</p>
+        <p>We want to understand how you navigate ubuntu.com, to improve our menus, and help you learn about our products more easily.</p>
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-nav">Join a navigation study</a></p>
+      </div>
+    </div>
+    <div class="row p-block">
+      <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3 ">
+        <hr class="p-rule">
+        <p class="p-heading--2">Ubuntu Security</p>
+        <p>We would love to learn how you manage your security and updates, and the way you use our CVE and security notices.</p>
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-security">Help us improve security</a></p>
+      </div>
+    </div>
+    <div class="row p-block">
+      <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3 ">
+        <hr class="p-rule">
+        <p class="p-heading--2">Ubuntu Pro</p>
+        <p>We are updating Ubuntu Pro with new features and coverage, to make it more powerful and intuitive for our customers.</p>
+        <p><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp-pro">Tell us how Ubuntu Pro works for you</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-muted-heading">How does it work?</h2>
+      </div>
+      <div class="col-9 col-medium-4 p-block">
+        <div class="row">
+          <div class="col-3">
+            <p class="p-heading--5"><a href="https://krzty8pox19.typeform.com/to/xZkdHbth#source=curp" class="is-black">1. Sign-up</a></p>
```

Comment:
It looks and feels wrong for headings to be links. It breaks the page consistency. I would suggest adding a descriptive link to this section instead.

Reply:
Any suggestions here @lyubomir-popov @cassiocassio ?